iChef-WebUi/src/app/app-routing.module.ts

```diff
@@ -1,12 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import {LandingPageComponent} from './components/landing-page/landing-page.component';
+import { RecipeDetailsComponent } from './components/recipe-details/recipe-details.component';
 import { RecipesComponent } from './components/recipes/recipes.component';
 
 const routes: Routes = [
     {path: '', redirectTo: '/home', pathMatch: 'full'},
     {path: 'home', component: LandingPageComponent},
-    {path: 'recipes', component: RecipesComponent}
+    {path: 'recipes', component: RecipesComponent},
+    {path: 'recipes/details', component: RecipeDetailsComponent}
```

Comment:
Can we have the id of the recipe in the url? We need this to be able to share links.